### ISSUE TITLE
Level pack names are untranslatable

### ISSUE THREAD

**Issue by gunchleoc** [Enigma-Game/Enigma#43]
... or in need of xgettext & push to Transifex? My translation is at 100%.

![enigma-levelpacks](https://user-images.githubusercontent.com/4095570/128239474-07d2c9b1-c51f-4f04-93b8-88501406f728.png)

Enigma version: 1.30 beta

**Comment by raoulb** [comment]
Yes, it might be the case that some strings are not translatable, either for technical reasons or because the code is incomplete.
This would need further investigation to find the string in the source and see if they are all wrapped in the correct i18n function calls.

**Comment by raoulb** [comment]
For example "Level Menu" here is not wrapped in an `_` function call (GNU gettext):

https://github.com/Enigma-Game/Enigma/blob/2d10fcaa984f017750f4bdcf44dcd07742db2111/src/gui/LevelMenu.cc#L439

I do not know, why.

**Comment by gunchleoc** [comment]
I managed to grep some:

```
src/gui/LevelPackConfig.cc:        set_caption(("Enigma - Level Pack Configuration"));
src/gui/LevelMenu.cc:        set_caption(("Enigma - Level Menu"));
src/gui/LevelPackConfig.cc:        Label * sokoballLabel = new Label("Sokoball:", HALIGN_RIGHT);
src/lev/RatingManager.cc:        return ecl::strf("%s+%d others", org.c_str(), others);

```

**Comment by raoulb** [comment]
Some of the static strings should be fixed by commit 7815d920339af7120fd94e02711fcd3fcacbad24.
